subprojects/internal-testing/src/main/groovy/org/gradle/util/TestPrecondition.groovy

```diff
@@ -97,9 +94,18 @@ enum TestPrecondition implements org.gradle.internal.Factory<Boolean> {
     JDK9_OR_EARLIER({
         JavaVersion.current() <= JavaVersion.VERSION_1_9
     }),
+    JDK9_OR_LATER({
+        JavaVersion.current() >= JavaVersion.VERSION_1_9
+    }),
+    JDK10_OR_LATER({
+        JavaVersion.current() >= JavaVersion.VERSION_1_10
+    }),
     JDK11_OR_EARLIER({
         JavaVersion.current() <= JavaVersion.VERSION_11
     }),
+    JDK11_OR_LATER({
+        JavaVersion.current() >= JavaVersion.VERSION_1_10
```

Comment:
Just curious, why `VERSION_1_10` and not `VERSION_10` like any of the following (and given 1.0 doesn't exist anymore in `JavaVersion`)

Reply:
`VERSION_1_10` is a constant in the public API. It was just done wrong back then. This is something we could clean up in the context of "tool chain capabilities".